MatterControl.Printing/PrinterCommunication/PrinterConnection.cs

```diff
@@ -1109,8 +1110,11 @@ public void Connect()
 									}
 									catch (ArgumentOutOfRangeException e)
 									{
-										TerminalLog.WriteLine("Exception:" + e.Message);
+										this.LogError("Exception:" + e.Message);
 										OnConnectionFailed(ConnectionFailure.UnsupportedBaudRate);
+
+										// TODO: Reimplement exception specific error messages
+										/*
```

Comment:
Create issue

Reply:
Issue created - MatterHackers/MCCentral#5766
Replace PrinterConnection MatterControl error messages with error and specific error codes